docker/pypi/dmwm-base/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM registry.cern.ch/cmsweb/cmsweb-base as cmsweb-base
-FROM registry.cern.ch/cmsweb/exporters as exporters
-FROM python:3.8-bullseye
+FROM registry.cern.ch/cmsweb/cmsweb-base:20230901-stable as cmsweb-base
```

Comment:
The timestamp for stable images is 2 years old, if I recall correctly the stable images retention policy is 1 year, maybe it was extended. I rather prefer to make extra effort and create new stable images with 202506xx-stable tag such that we'll perform major migration ensuring that stable images will not disappear suddenly.

Reply:
ah, maybe i should look at the `cmsweb/cmsweb-alma9-base` image, give me a sec

Reply:
the latest tag for `cmsweb/cmsweb-alma9-base` is `20250506` without any `-stable` suffix. for the time being i would stick with this tag

Reply:
Dario, I rather to not use non stable image since their retention policy is quite short, if I recall correctly it is about 3 months. Instead, just do the following:
```
docker pull registry.cern.ch/cmsweb/cmsweb-alma9-base:20250506
docker tag registry.cern.ch/cmsweb/cmsweb-alma9-base:20250506 registry.cern.ch/cmsweb/cmsweb-alma9-base:20250506-stable
docker push registry.cern.ch/cmsweb/cmsweb-alma9-base:20250506-stable
```
and then use stable image. This will create an alias tag and push it to registry where new image will be hold for longer period of time.

Reply:
I know i know, i just do not have permissions to push to `cmsweb-alma9-base`

Reply:
but ok, assuming you or aroosha will do it, i changes the tag to use the `-stable`

Reply:
done:
```
docker push registry.cern.ch/cmsweb/cmsweb-alma9-base:20250506-stable
The push refers to repository [registry.cern.ch/cmsweb/cmsweb-alma9-base]
bb19aa74b58c: Layer already exists
b92fcbc8bdec: Layer already exists
1e4ed39e79fd: Layer already exists
84bb803a6a3c: Layer already exists
ec623c3d76de: Layer already exists
4480616e9270: Layer already exists
20250506-stable: digest: sha256:6f910ff0a84960fba4f58f4df5e60fa88658ea95f7bb2f362a864ec70e89625f size: 1582
```

Reply:
thanks, then i mark this conversation as resolved